docs/guidesdraft/01-getting-started.md

```diff
@@ -0,0 +1,138 @@
+## Getting started
+
+### Introduction
+
+[CARTO VL](https://github.com/cartodb/carto-vl) is a JavaScript library to create custom location intelligence applications that leverage the power of [CARTO](https://carto.com/). It uses [WebGL](https://www.khronos.org/webgl/) to enable powerful vector maps.
+
+It relies on [Mapbox GL](https://www.mapbox.com/mapbox-gl-js/api) to render the basemaps and it can be used for other functionality too, read [Mapbox GL documentation](https://www.mapbox.com/mapbox-gl-js/api/) for more information. However, keep in mind that CARTO VL layers can only be controlled with the CARTO VL API (this documentation) and that Mapbox GL native layers can only be controlled with the Mapbox GL API. Therefore, CARTO VL expressions cannot be used for Mapbox GL layers and vice versa.
```

Comment:
IMO it is not very clear which is "the second it". I would rephrase to something like: "It relies on Mapbox GL to render the basemaps but that library can also be used to complement CARTO VL with some other functionalities (read Mapbox GL documentation for more information)"